commands/bench/lib/bench-all-trappist.sh

```diff
@@ -14,7 +14,7 @@ chain="${runtime}-dev"
 export RUST_LOG="${RUST_LOG:-error}"
 
 echo "[+] Compiling benchmarks..."
-cargo build --profile $profile --locked --features=runtime-benchmarks
+cargo build --profile $profile --locked --features=runtime-benchmarks,riscv
```

Comment:
I have no idea what this is building as no `-p` is supplied. But just remove this as whatever trappist is has no riscv feature probably.